VotingApplication/VotingApplication.Web/Api/Controllers/ManageController.cs

```diff
@@ -140,7 +155,30 @@ public void Put(Guid manageId, ManagePollUpdateRequest updateRequest)
                     newOptions.AddRange(updateRequest.Options);
                 }
 
+                List<Token> redundantTokens = existingPoll.Tokens.ToList<Token>();
 
+                foreach (Token voter in updateRequest.Voters)
+                {
+                    if (voter.TokenGuid == Guid.Empty)
```

Comment:
This feels a bit hackish. Are we using the Data.Model.Token here because of a specific reason, or because it's defined and convenient. Checking for an empty guid seems like it would make more sense if it could be checking a nullable guid for hasValue?